insitu_utils/lib/insitu_utils/painter.hpp

```diff
@@ -157,6 +157,11 @@ class Painter
                            const std::string& str,
                            int face = cv::FONT_HERSHEY_PLAIN, int thickness = 1,
                            cv::Scalar color = Color::white);
+    
+    static void drawtorect_multiline(cv::Mat& mat, cv::Rect target,
+                           const std::string& topic_name, const std::string& str, int lines,
+                           int face = cv::FONT_HERSHEY_PLAIN, int thickness = 1,
+                           cv::Scalar color = Color::white);
```

Comment:
This is the number of lines, correct? Could be changed to be more descriptive
```suggestion
    static void drawtorect_multiline(cv::Mat& mat, cv::Rect target,
                           const std::string& topic_name, const std::string& str, int num_lines,
                           int face = cv::FONT_HERSHEY_PLAIN, int thickness = 1,
                           cv::Scalar color = Color::white);
```